More correct monster names

Hypexion opened the pull request:
Part two of my crusade to make names more consistent. This PR covers Fegundo, Seep, Rock Golems and Rock Giants.

This doesn't cover any of the snakes, or engineers, because they're all internally consistent. Also, the snake file names make it pretty obvious what lives inside them.

mrimer commented:
@Hypexion , would you please rebase this PR?

mrimer commented on DRODLib/RockGiant.cpp:
```diff
@@ -20,11 +20,15 @@
  *
  * ***** END LICENSE BLOCK ***** */
 
+//Splitter.cpp
```
Remove this comment.

mrimer commented on DRODLib/RockGiant.h:
```diff
@@ -20,33 +20,43 @@
 
-#ifndef ROCKGOLEM_H
-#define ROCKGOLEM_H
+//Splitter.h
```
Clean up this comment.

mrimer reviewed (changes requested):
Looks good!
Thank you for the good neighborly effort to keep the codebase clean.
I suggested minor tweaks.

The PR says that Texts/Speech.uni has a conflict.  Would you please resolve this, and then I'll merge.  (Sorry, I don't know of an easy way to do that.  Do you have a utility that can perform a readable diff on the .uni files?)

mrimer commented:
Oh, and may I request to update the DRODLib.2019.vcxproj project file (etc) with the new file names?

Hypexion commented:
@mrimer I've updated the VS2019 project file, fixed up the comments and resolved the conflict.

It appears that most Git clients don't recognise the UCS2 encoding the uni files use as text, which is certainly a decision they are allowed to make. Luckily, TortioseGit does treat them as text, allowing you merge and diff them normally. Changing the encoding might help, but since a free tool exists that works, it's probably safer to leave it.
